recipes/opencolorio/all/patches/2.3.1-0001-fix-cmake-source-dir-and-targets.patch

```diff
@@ -68,7 +68,7 @@ index 26b4bb4c..d54cff3d 100755
          "$<BUILD_INTERFACE:xxHash>"
 -        ${YAML_CPP_LIBRARIES}
 -        MINIZIP::minizip-ng
-+        yaml-cpp
++        PUBLIC yaml-cpp
```

Comment:
I am bringing this back, as it is related to a limitation of CMakeDeps, sorry for the noise.
There is already some work in a new improved ``CMakeDeps`` generator that will make this unnecessary.